keyboards/satan/keymaps/abhixec/keymap.c

```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
+
+
+
+// Used for SHIFT_ESC
```

Comment:
Your keymap isn't using the functionality provided by Lines 5-6. They can be safely deleted.